.github/ISSUE_TEMPLATE/prepare_release.md

```diff
@@ -14,14 +14,49 @@ For detailed info on the release process refer to https://github.com/waku-org/nw
-- [ ] generate and edit releases notes
-- [ ] open PR and merge release notes to master
-- [ ] cherry-pick release notes to the release branch
-- [ ] assign release tag to the cherry-picked release notes commit
-- [ ] create GitHub release
-- [ ] deploy the release to DockerHub
-- [ ] [deploy release to `waku.sandbox` fleet](https://ci.infra.status.im/job/nim-waku/job/deploy-waku-sandbox)
-- [ ] announce the release
+
+- [ ] Create release branch
+- [ ] Assign release candidate tag to the release branch HEAD. e.g. v0.30.0-rc.0
+- [ ] Generate and edit releases notes
+- [ ] _End user impact_: Summarize impact of changes on Status end users (can be a comment in this issue).
+- [ ] **Validate release candidate**
+
+  - [ ] **On Waku fleets**
+    - [ ] Lock `waku.test` fleet to release candidate version
+    - [ ] Continuously stress `waku.test` fleet for a week (e.g. from `waku-dev`)
+    - [ ] Search _Kibana_ logs from the previous month (since last release was deployed), for possible crashes or errors in `waku.test` and `waku.sandbox`.
+      - Most relevant logs are `(fleet: "waku.test" OR fleet: "waku.sandbox") AND message: "SIGSEGV"`
+    - [ ] Run release candidate with `waku-simulator`, ensure that nodes connected to each other
+    - [ ] Unlock `waku.test`
+
+  - [ ] **On Status fleet**
+    - [ ] Deploy release candidate to `status.staging`
+    - [ ] Perform [sanity check](https://www.notion.so/How-to-test-Nwaku-on-Status-12c6e4b9bf06420ca868bd199129b425)
+      - [ ] Connect 2 instances to `status.staging` fleet
+      - [ ] Send contact request across and accept
+      - [ ] 1:1 Chats with each other
+      - [ ] Send and receive messages in a community
+      - [ ] Close one instance, send messages with second instance, reopen first instance and confirm messages sent while offline are retrieved from store
+    - [ ] Perform checks based _end user impact_
+    - [ ] Ask other CCs to point their instance to `status.staging` for a week and use the app as usual.
+    - [ ] Ask Status/QA to perform sanity checks (as described above) + checks based on _end user impact_
+    - [ ] Get other CCs sign-off: they comment on this PR "used app for a week, no problem", or problem reported, resolved and new RC
+    - [ ] Get Status/QA sign-off
+
+- [ ] **Proceed with release**
+
+  - [ ] Assign a release tag to the same commit that contains the validated release-candidate tag
+  - [ ] Create GitHub release
+  - [ ] Deploy the release to DockerHub
+  - [ ] Announce the release
+  - [ ] Submit a PR from the release branch to master. Important to commit the PR with "create a merge commit" option.
+
+- [ ] **Promote release to fleets**.
+  - [ ] [Deploy final release to `waku.sandbox` fleet](https://ci.infra.status.im/job/nim-waku/job/deploy-waku-sandbox)
+  - [ ] Deploy final release to `waku.prod`
+  - [ ] Deploy final release to `status.staging`
+  - [ ] Deploy final release to `status.test` (soon to be `status.prod`)
```

Comment:
Should we add CI links for all these steps?
Or do the last steps means pinging infra?

Reply:
Yes, good point!

Links:
- `waku.test` - https://ci.infra.status.im/job/nim-waku/job/deploy-waku-test/
- `waku.sandbox` - https://ci.infra.status.im/job/nim-waku/job/deploy-waku-sandbox/
- `shards.staging` - https://ci.infra.status.im/job/nim-waku/job/deploy-shards-staging/
- `shards.test` - https://ci.infra.status.im/job/nim-waku/job/deploy-shards-test/

Reply:
won't nwaku master get deployed automatically to `waku.test`? isn't it how ti works?

Reply:
> won't nwaku master get deployed automatically to `waku.test`? isn't it how ti works?

Yes, you are absolutely right. Nevertheless, during the testing time, we disable the automatic deployments and leave the release candidate version running for some days (ideally one week.)

Reply:
> Yes, you are absolutely right. Nevertheless, during the testing time, we disable the automatic deployments and leave the release candidate version running for some days (ideally one week.)

Yes, we have an entry `unlock waku.test` already, so I assume the deployment will resume. Let me know if it's not clear enough in the new tempalte.